lib/OpenQA/Utils.pm

```diff
@@ -1313,5 +1313,30 @@ sub any_array_item_contained_by_hash {
     return 0;
 }
 
+#replace %NAME% with $settings{NAME}
+sub reset_settings {
```

Comment:
I'm wondering about this function name. Wouldn't `expand_settings` be much better?